pallets/capacity/src/types.rs

```diff
@@ -93,17 +93,32 @@ impl<T: Config> StakingAccountDetails<T> {
 	}
 
 	/// Decrease the amount of active stake by an amount and createa an UnlockChunk.
-	pub fn decrease_by(&mut self, amount: BalanceOf<T>, thaw_at: T::EpochNumber) -> DispatchResult {
-		let new_active = self.active.saturating_sub(amount);
-
-		let unlock_chunk = UnlockChunk { value: amount, thaw_at };
-
-		self.active = new_active;
+	pub fn deduct(
```

Comment:
Suggestion only: In my head `withdraw` would be the antonym to `deposit`.
```suggestion
	pub fn withdraw(
```